backend/copr_backend/pulp.py

```diff
@@ -178,3 +188,42 @@ def delete_distribution(self, distribution):
         """
         url = self.config["base_url"] + distribution
         return requests.delete(url, **self.request_params)
+
+    def list_packages(self, repository_version):
+        """
+        List packages for a given repository version
+        Ideally, we would list packages for a repository but that probably
+        isn't possible.
```

Comment:
Doesn't the `response["results"]["packages"]` return what we want?

Reply:
After PR #3391 this function wasn't needed, so I removed it :-)